core/tests/test_trezor.protobuf.py

```diff
@@ -1,14 +1,14 @@
 from common import *
 
 from trezor import protobuf
-from trezor.messages import WebAuthnCredential, Failure, SignMessage, DebugLinkMemoryRead
+from trezor.messages import AuthorizeCoinJoin, WebAuthnCredential, Failure, SignMessage, DebugLinkMemoryRead
 
 
 def load_uvarint32(data: bytes) -> int:
     # use known uint32 field in an all-optional message
     buffer = bytearray(len(data) + 1)
     buffer[1:] = data
-    buffer[0] = (1 << 3) | 0  # field number 1, wire type 0
+    buffer[0] = (1 << 3) | 7  # field number 1, wire type 7
```

Comment:
I don't think this change is appropriate? the "wire type" is from protobuf spec, 0 means "varint". The enum values we use internally have nothing to do with this.